cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java

```diff
@@ -372,33 +374,34 @@ public void testService() throws Exception {
       testCommandOutputContains(cli, "call service " + qualifiedServiceIdV1
         + " POST /echo body \"testBody\"", ":testBody");
     } finally {
-      testCommandOutputContains(cli, "stop service " + qualifiedServiceId, "Successfully stopped service");
-      testCommandOutputContains(cli, "stop service " + qualifiedServiceIdV1, "Successfully stopped service");
-      assertProgramStatus(programClient, service, "STOPPED");
-      assertProgramStatus(programClient, serviceV1, "STOPPED");
+      // Stop all running services
+      programClient.stopAll(NamespaceId.DEFAULT.toId());
     }
   }
 
   @Test
   public void testRuntimeArgs() throws Exception {
     ServiceId service = fakeAppId.service(PrefixedEchoHandler.NAME);
     ServiceId serviceV1 = fakeAppIdV1.service(PrefixedEchoHandler.NAME);
-    String qualifiedServiceId = String.format("%s.%s", FakeApp.NAME, PrefixedEchoHandler.NAME);
+    String qualifiedServiceId = String.format("%s.%s.%s", FakeApp.NAME, ApplicationId.DEFAULT_VERSION,
```

Comment:
Service id in the format <app-id.program-id> has different behavior from  <app-id.DEFAULT_VERSION.program-id>, when multiple versions exist, see my comment in `ServiceClient.java`.  I specify the default version to guaranteed that the service routed to the default version.